pkg/agent/agent_test.go

```diff
@@ -1966,3 +2037,66 @@ func updateActionToNode(t *testing.T, action k8stesting.Action) *corev1.Node {
 
 	return node
 }
+
+// lifted from https://github.com/kubernetes/kubectl/blob/master/pkg/drain/drain_test.go.
```

Comment:
I wonder why linter did not complain about this :thinking: 
```suggestion
// Lifted from https://github.com/kubernetes/kubectl/blob/master/pkg/drain/drain_test.go.
```